### PR TITLE
porting radio inputs to Lit

### PR DESCRIPTION
**Pull request by dlockhart**
Radio inputs are here! These are a bit different, since we can't do a custom element as inputs in the same group can't span shadow roots. So just styles this time around, which is probably what I'll do for `<select>`s as well.

### CODE REVIEW

File: components/inputs/README.md [hunk 1]
```diff
@@ -180,3 +181,68 @@ class MyElem extends LitElement {
 
 }
 ```
+
+## Radio Buttons
+
+Unlike checkboxes, individual radio buttons cannot be placed in a custom element. Items belonging to a radio group cannot span across different shadow roots -- all radios in the same group must be in the same shadow root.
```
Comment (dbatiste) [inline components/inputs/README.md]: If we think there is value in wrapping, for consistency or ease of use, we _could_ provide a component that renders to the light-DOM. The node into which the template is rendered can be overridden by implementing [createRenderRoot](https://lit-element.polymer-project.org/guide/templates#renderroot)... Lit-element defaults to creating an open shadow root:
```
class InputRadio extends LitElement {
   createRenderRoot() {
      return this;
   }
}
```

Comment (dlockhart) [reply components/inputs/README.md]: Oh _interesting_, I didn't consider that. 🤔

File: components/inputs/test/input-radio.visual-diff.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+	});
+
+	async function getRadio(id, val) {
```
Comment (dbatiste) [inline components/inputs/test/input-radio.visual-diff.js]: I think the `async` and the `await` are not needed here.  The `getRadio` is returning a promise provided by `evaluateHandle`, and the consumer of `getRadio` is in control of whether to wait or not (which it is).

File: components/inputs/README.md (hunk 1 above)
Comment (mdgbayly) [reply components/inputs/README.md]: I think we tried something similar to that in Polymer 3, prior to going the style module route. I think there were limitations of using light DOM which I tried to document the pros/cons of in this PR https://github.com/BrightspaceUI/inputs/pull/51.
Based on that we decided to go with a style module.
Would need to re-read that lengthy PR and absorb/compare to Lit, to determine if the problems highlighted there would still be an issue with Lit...

Comment (dlockhart) [reply components/inputs/README.md]: Thanks for that link Martin. I tried this out last week and basically ran into all the issues you call out there -- Lit styles stopped working and no `<slot>` support. As you did, I was able to get styles working by moving them into the component itself, but for the same reasons I don't love that approach.

I think I'm leaning towards abandoning having a `<d2l-input-radio>` component and just relying on folks importing the style and applying it to native radios, which is what this PR does. At least now with Lit we have a nicer way to share these styles.

**Pull request merged**